src/registrar/admin.py

```diff
@@ -2552,6 +2552,33 @@ def save_model(self, request, obj, form, change):
         super().save_model(request, obj, form, change)
 
 
+class PortfolioAdmin(ListHeaderAdmin):
+    # NOTE: these are just placeholders.  Not part of ACs (haven't been defined yet).  Update in future tickets.
+    list_display = ("organization_name", "federal_agency", "creator")
+    search_fields = ["organization_name"]
+    search_help_text = "Search by organization name."
+    # readonly_fields = [
+    #     "requestor",
+    # ]
+
+    def save_model(self, request, obj, form, change):
+        # ---- update creator ----
+        # Set the creator field to the current admin user
+        obj.creator = request.user if request.user.is_authenticated else None
+
+        # ---- update organization name ----
+        # org name will be the same as federal agency, if it is federal,
+        # otherwise it will be the actual org name. If nothing is entered for
+        # org name and it is a federal organization, have this field fill with
+        # the federal agency text name.
+        is_federal = obj.organization_type == DomainRequest.OrganizationChoices.FEDERAL
```

Comment:
**blocking** this is very clean. Instead, move organization_type to a shared contents file like what was recently done with federal_type.

Reply:
(we discussed in huddle that we will do this in a separate ticket)